doc/contributing.md

```diff
@@ -0,0 +1,94 @@
+# Contributing a module to *Oecophylla*
```

Comment:
Is this capitalization OK and respected in other places in the documentation?

Reply:
Do we have a 'house style' for capitalization?

Reply:
No? But I think this is the only place where the O is capitalized ...

Reply:
BTW, could you also check the README so it follows this?

Reply:
👍